src/views/Home.jsx

```diff
@@ -48,16 +48,42 @@ export function Home({ data, setListPath }) {
 				/>
 				<button type="submit">Create a list</button>
 			</form>
-			<ul>
-				{data.map((item) => (
-					<SingleList
-						key={item.path}
-						name={item.name}
-						path={item.path}
-						setListPath={setListPath}
-					/>
-				))}
-			</ul>
+			{data.length ? (
+				<ul>
+					<p className="list-title">Your Lists</p>
+					{data.map((item) => (
+						<SingleList
+							key={item.path}
+							name={item.name}
+							path={item.path}
+							setListPath={setListPath}
+						/>
+					))}
+				</ul>
+			) : (
+				<div>
+					<p>
+						There are no lists to display. Please create a list to get started!
+					</p>
+				</div>
+			)}
+			{/* <div className="home-links">
```

Comment:
Just a nit: will we keep this here? I saw the same code further down. If not, we can also delete 3 import lines from the file:
```
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faSquareGithub } from '@fortawesome/free-brands-svg-icons';
import StackedLogo from './StackedLogo.jsx';
```

Reply:
And in response to what Jose has said, I will experiment with the loading bars and convene with Rachel with what we think we want to do about the delays. Also Jose and Iryna we can talk about what we'd like to do about the delays as well. Thanks!